File: 06_상속/src/com/javastudy/Car.java
package com.javastudy;

public class Car {
	
	private String name;
	private int len;
	
	public Car(String name, int len) {
		this.name = name;
		this.len = len;
	}
	
	public void prnCar() {
		System.out.println("c.j.이름 : " + name);
		System.out.println("c.j.거리 : " + len);
	}
}

File: PythonP/06_상속/com/pythonstudy/Car.py
'''
Created on 2022. 5. 26.

@author: admin
'''

class Car:
    def __init__(self, name, len):
        self.name = name
        self.len = len
    
    def prnCar(self):
        print(self.name)
        print(self.len)